generators/client-2/templates/src/main/webapp/app/_app.ng2module.ts

```diff
@@ -26,12 +27,12 @@ import { ActiveMenuDirective } from './layouts/navbar/active-menu.directive';
         <%=angular2AppName%>AccountModule
     ],
     declarations: [
+        <%=jhiPrefixCapitalized%>LoginModalComponent,
```

Comment:
Do you mean export it from app module and then import it in account module from the app module ? What about home and navbar components do i import it directly in the components ?